hrdcdb/app/routes.py

```diff
@@ -229,7 +230,44 @@ def add_service(clientid):
 	if form.validate_on_submit():
 		form.execute_transaction()
 		return redirect(url_for('add_service', clientid = clientid))
-	return render_template('add_service.html', title = 'Add Service', form = form, data = services, cid = clientid)
+	#return render_template('add_service.html', title = 'Add Service', form = form, data = services, cid = clientid)
+
+	elif request.method == 'GET':
+
+		# check for request parameters
+		if request.args.get('recordID'):
+
+			#get the recorde id that needs to be deletedf
+			recordID = request.args.get('recordID')
+
+			# update database
+			recordToDel = Service.query.filter(Service.id == recordID).all()
```

Comment:
use .first() instead of .all(), it just returns a single item rather than a list of one